mappings/net/minecraft/util/LongObjectHashMap.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_101 net/minecraft/util/LongObjectHashMap
+	FIELD field_324 entries [Lnet/minecraft/class_102;
```

Comment:
Would make sense to also map out class_102 which would be LongObjectHashMapEntry, its very similar to IntHashMapEntry

Reply:
Can also look at IntHashMap for the unmapped methods here and also making the names match between them for consistency